xAPI.md

```diff
@@ -438,6 +439,16 @@ __Registration__: An instance of a learner experiencing a particular Activity.
 __Representational State Transfer (REST)__: An architecture for designing networked web Services.
 It relies on HTTP methods and uses current web best practices.
 
+<a name="def-reporting-system" />
+
+__Reporting System__: A system or tool used to report on and analyse xAPI data and display this to the user. 
+Within this specification, requirements relating to the display of xAPI data to a user are often directed
+at reporting systems. Note that debugging tools used by developers to display data during development
```

Comment:
In theory, yes, but I don't think we need to be exhaustive. Happy to add it if you feel strongly but I'm interpreting your smiley face as not feeling strongly!